docs/decisions/client-repo-mediawiki.md

```diff
@@ -0,0 +1,15 @@
+# Client / Repo relation to MediaWiki
+
+**Decision Date:** Early 2021
```

Comment:
I'd put January 2021 here.
The only reason I was general about the previous decisions was that it didn't seem to make sense to spend the time figuring out the exact month we decided.

Reply:
done